src/ReactApp.test.js

```diff
@@ -1,8 +1,9 @@
-/* eslint-disable linebreak-style */
-import { render, screen } from '@testing-library/react';
-import App from './ReactApp';
+/* eslint-disable import/extensions */
+/* eslint-disable quotes */
```

Comment:
- [ ] Kindly make sure you fix all the linters errors everywhere in your code base you disable them for clean code and best practices reasons.